source/Integrate/Open_Source_Apps/index.html

```diff
@@ -38,5 +38,11 @@
         <span class="icon-wordpress"></span>
       </a>
     </li>
+       <li>
+      <a href="modx.html">
+        MODX
+        <span class="icon-modx"></span>
```

Comment:
I saw you are using the icomoon fontpack, which doesn't have the MODX logo included. The FontAwesome pack does.

I only have the logo as an transparent PNG:

![modx](https://cloud.githubusercontent.com/assets/487567/12670714/e5028d54-c66a-11e5-9cf8-ef0129cd2ea9.png)